### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by jamoralp**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,18 +6,22 @@
...
-[![Compare Images](https://images.microbadger.com/badges/image/eprosima/micro-xrce-dds-agent.svg)](https://microbadger.com/images/eprosima/micro-xrce-dds-agent/)
+[![Docker Build Status](https://img.shields.io/docker/cloud/build/eprosima/micro-xrce-dds-agent)](https://hub.docker.com/r/eprosima/micro-xrce-dds-agent/)
 
 <a href="http://www.eprosima.com"><img src="https://encrypted-tbn3.gstatic.com/images?q=tbn:ANd9GcSd0PDlVz1U_7MgdTe0FRIWD0Jc9_YH-gGi0ZpLkr-qgCI6ZEoJZ5GBqQ" align="left" hspace="8" vspace="2" width="100" height="100" ></a>
 
-*Micro XRCE-DDS Agent* acts as a server between DDS Network and *Micro XRCE-DDS Clients*.
-*Micro XRCE-DDS Agents* receive messages containing Operations from Clients.
-Agents keep track of the Clients and the *Micro XRCE-DDS Entities* they create.
-The Agent uses the Entities to interact with DDS Global Data Space on behalf of the Client.
+*Micro XRCE-DDS Agent* acts as a server between the DDS dataspace network and *Micro XRCE-DDS Client* applications.
```
Comment (FranFin) [inline README.md:13]:
```suggestion
*The Micro XRCE-DDS Agent* acts as a server to bridge the DDS dataspace network with *Micro XRCE-DDS Client* applications.
```

File: README.md [hunk 2]
```diff
@@ -6,18 +6,22 @@
...
 
-The communication between a *Micro XRCE-DDS Client* and a *Micro XRCE-DDS Agent* supports two kind transports: UDP or SerialPort.
-While running *Micro XRCE-DDS Agent* will attend any received request from your *Micro XRCE-DDS Clients*. *Micro XRCE-DDS Agent* answers back with the result of a request each time a request is attended.
+*Micro XRCE-DDS Agents* receive messages containing request operations from *Clients* to publish and subscribe to/from the DDS global dataspace.
```
Comment (FranFin) [inline README.md:15]:
```suggestion
The *Micro XRCE-DDS Agents* receive messages containing request operations from the *Clients* to publish and subscribe to topics in the DDS global dataspace. Remote procedure calls, as defined by the DDS-RPC standard, are also supported, allowing to communicate according to a request/reply paradigm.
```

File: README.md [hunk 3]
```diff
@@ -6,18 +6,22 @@
...
+The *Agents* then process these XRCE requests and send back a response with the operation status result and with the requested data, in the case of subscribe operations.
```
Comment (FranFin) [inline README.md:16]:
```suggestion
The *Agents* then process these XRCE requests and send back a response with the operation status result and with the requested data, in the case of subscribe/request/reply operations.
```

File: README.md [hunk 4]
```diff
@@ -6,18 +6,22 @@
...
+Remote procedure calls, as defined by the DDS-RPC standard, are also supported.
```
Comment (FranFin) [inline README.md:17]:
```suggestion
```

File: README.md [hunk 5]
```diff
@@ -6,18 +6,22 @@
...
+
+*Agents* keep track of the *Clients* and the *Micro XRCE-DDS Entities* they create, by means of a dedicated *ProxyClient* entity that acts on behalf of them.
```
Comment (FranFin) [inline README.md:19]:
```suggestion
*Agents* keep track of the *Clients* by means of a dedicated *ProxyClient* entity that acts on behalf of the latter.
```

File: README.md [hunk 6]
```diff
@@ -6,18 +6,22 @@
...
+To do so, the *Agent* uses the *XRCE Entities* to interact with the DDS Global dataspace.
```
Comment (FranFin) [inline README.md:20]:
```suggestion
This is made possible by the creation of *DDS Entities* on the *Agent* as a result of *Clients*' operations, such as *Participants*, *Topics*, *Publishers*, and *Subscribers*, which can interact with the DDS Global dataspace.
```

File: README.md [hunk 7]
```diff
@@ -6,18 +6,22 @@
...
+
+The communication between a *Micro XRCE-DDS Client* and a *Micro XRCE-DDS Agent* is achieved by means of several kinds of built-in transports: **UDPv4**, **UDPv6**, **TCPv4**, **TCPv6** and **Serial** communication.
```
Comment (FranFin) [inline README.md:22]:
```suggestion
The communication between a *Micro XRCE-DDS Client* and a *Micro XRCE-DDS Agent* is achieved by means of several kinds of built-in transports: **UDPv4**, **UDPv6**, **TCPv4**, **TCPv6** and **Serial** communication. In addition, there is the possibility for the user to generate its own **Custom** transport.
```

File: README.md [hunk 8]
```diff
@@ -6,18 +6,22 @@
...
+You can use an *Agent* with these transports by means of the standalone executable generated when building the project, which comes with a built-in CLI tool to select one among the list previously mentioned.
```
Comment (FranFin) [inline README.md:23]:
```suggestion
You can use an *Agent* with these transports by means of the standalone executable generated when building the project, which comes with a built-in CLI tool to select one of the transports listed above.
```

**Review by FranFin — CHANGES_REQUESTED** [review]

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Agent%20Manual%20Linux/356/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Agent%20Manual%20Windows/357/)

**Comment by jamoralp** [comment]
@FranFin ready to merge :)

**Review by pablogs9 — APPROVED** [review]

**Review by FranFin — APPROVED** [review]

**Pull request merged**